src/olympia/amo/models.py

```diff
@@ -378,10 +378,10 @@ def save(self, **kwargs):
         return super(SaveUpdateMixin, self).save(**kwargs)
 
 
-class ModelBase(SearchMixin, caching.base.CachingMixin, SaveUpdateMixin,
-                models.Model):
+class UncachedModelBase(SearchMixin, SaveUpdateMixin, models.Model):
```

Comment:
should `manual_order` be in this class rather than `ModelBase`?

Reply:
`manual_order` is a function, not a method, living outside any class. Yes, it's confusing and weird.